lib/web_ui/lib/src/engine/semantics/checkable.dart

```diff
@@ -99,7 +99,7 @@ class Checkable extends RoleManager {
   }
 
   void _updateDisabledAttribute() {
-    if (!semanticsObject.hasFlag(ui.SemanticsFlag.isEnabled)) {
+    if (!semanticsObject.isEnabled) {
```

Comment:
In theory the framework allows you to specify a checkbox that is neither marked as enabled or disabled (hasEnabledState not set). This logic will mark those incorrectly as disabled.

Reply:
correct, I changed the logic. thanks :)